src/SFA.DAS.ApprenticeCommitments/Application/Commands/ChangeApprenticeshipCommand/ChangeApprenticeshipCommandHandler.cs

```diff
@@ -0,0 +1,50 @@
+﻿using FluentValidation;
+using MediatR;
+using SFA.DAS.ApprenticeCommitments.Data;
+using SFA.DAS.ApprenticeCommitments.Data.Models;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Application.Commands.ChangeApprenticeshipCommand
+{
+    public class ChangeApprenticeshipCommandValidator : AbstractValidator<ChangeApprenticeshipCommand>
```

Comment:
Generally the general practice in the SFA is to put these into separate files (but it's up to you).